pom.xml

```diff
@@ -448,7 +448,7 @@
                                 <configuration>
                                     <sources>
                                         <source>${project.basedir}/src/main/301until320-all/scala</source>
-                                        <source>${project.basedir}/src/main/301until320-nondb/scala</source>
+                                        <source>${project.basedir}/src/main/311until320-all/scala</source>
```

Comment:
how is this working?  isn't is missing Spark30XShims?

Reply:
it does not work, it's an incorrect merge conflict resolution. and this build is not covered by pre-merge

Reply:
ah, ok.